application/menunode.class.inc.php

```diff
@@ -273,12 +274,23 @@ public static function GetMenuGroups($aExtraParams = array())
 				continue;
 			}
 
+			$aSubMenuNodes = static::GetSubMenuNodes($sMenuGroupIdx, $aExtraParams);
+			if (! ParentMenuNodeCompiler::$bUseLegacyMenuCompilation && !($oMenuNode instanceof ShortcutMenuNode)){
+				if (is_array($aSubMenuNodes) && 0 === sizeof($aSubMenuNodes)){
```

Comment:
```suggestion
				if (is_array($aSubMenuNodes) && 0 === count($aSubMenuNodes)){
```

Usage of `count()` if prefered over `sizeof()` as the later as a different meaning in other languages so it avoids confusion. Also it's an alias of `count()` so it might be deprecated.